site/ts/twig/grading/electronic/AutogradingPanel.ts

```diff
@@ -53,5 +60,11 @@ $(() => {
     $('#autograding-results-regrade-all').on('click', () => {
         regrade(0, HIGHEST_VERSION, GRADEABLE_ID, USER_ID);
     });
+    $('.dropdown-item').on('click', () => {
```

Comment:
It may be better to change this to a different class with a class named uniquely to the autograding panel regrade dropdown menu buttons (since the dropdown-item class is used for styling on other pages)